xml/templates/drop.tpl

```diff
@@ -11,7 +11,7 @@
 --{/if}
 -- /*******************************************************
 -- *
--- * Clean up the existing tables{if isset($no_license) and $no_license} - this section generated from {$smarty.template}
+-- * Clean up the existing tables{if !$no_license} - this section generated from {$smarty.template}
```

Comment:
I think this line is backwards from the original - should be `if $no_license` ?

Reply:
if not it doesn't have no license.

Reply:
@demeritcowboy frigg - I'm gonna kill no_license - sane variable name on the way!

Reply:
oh IIIIII ain't got no no_licences, no license , no license ain't got me